decentraland/unity-renderer
# Fix: allow scrolling views when overing on carousels

## What does this PR change?

Fix #4217
Fixes the scrolling of scroll views while overing on carousels

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?explorer-branch=fix/scroll-view-carousel
2. Open a passport
3. Click on the collectibles tab
4. Try scrolling the view and check that it works when the mouse is over a carousel

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
